pypeman/channels.py

```diff
@@ -480,6 +495,51 @@ def graph_dot(self, end=''):
             for entry in sub.graph_dot(end=end):
                 yield entry
 
+    def _init_callback_nodes(self, *callback_nodes):
+        callback_nodes = [node for node in flatten(callback_nodes) if node]
+        for node in callback_nodes:
+            node.channel = self
+
+        if len(callback_nodes) > 1:
+            previous_node = callback_nodes[0]
+
+            for node in callback_nodes[1:]:
+                previous_node.next_node = node
+                previous_node = node
+        return callback_nodes
```

Comment:
function name seems a little confusing.
it seems to do many things and not all are related to callbacks. but perhaps I'm missing some context